app/main/download_data.py

```diff
@@ -8,26 +8,26 @@
 
 
 area = {
-    "name": "options",
+    "name": "area",
```

Comment:
just checking, do we expect this kind of data to be in the upcoming release? I don't know if the model / data is robust enough to provide this currently. Presumably the idea it gets looked up from a postcode, but which post-code (in the data ingest form) should we use? They are not in a consistent formats it is:

- field Organisation is free text without any kind of mapping, and so open to all kind of spelling/ mismatch errors
- No Location or postcode is provided with this field on the form
- Each form can have single or multiple postcodes mapped to each project - there is no validation or the text field for these, so it would be difficult to extract automatically

Reply:
I think we should aim for this format, if projects don't have a postcode then they should be filtered out, I guess if they have multiple postcodes if one of the postcodes is in the region it should be included?